ezcontent_publish.permissions.yml

```diff
@@ -1,3 +1,6 @@
 ezcontent publish view user profile:
   title: 'View user profile'
   description: 'Allow user to view profile tab.'
+ezcontent user based shortcut:
```

Comment:
add this permission for editor role in config, also please see the ticket point 2 

2. **Permission for roles to enable automatic user shortcut generation on user login.** (You are not creating shortcut set itself, where as requirement says the default shortcut should not generate if role does not have permission.)

Please update the title and description accordingly.